### ISSUE TITLE
Cannot run project

### ISSUE THREAD

**Issue by wallaceferreira** [angular/universal#185]
Hi

I've been trying to run the universal project but I'm unable to make it work. I'm not sure if it's some missing configuration or other issue.

That's what I've done:

1) Cloning the repo:  **git clone https://github.com/angular/universal.git**
2) Installing via npm: **npm install**
3) Running: **npm run serve**

Steps 1) and 2) work just fine. However,  the process fails at step 3 with this error:

```
> angular2-universal@0.0.1 serve /scratch/universal2/universal
> gulp serve

[12:44:02] Using gulpfile /scratch/universal2/universal/gulpfile.js
[12:44:02] Starting 'watch'...
[12:44:02] Finished 'watch' after 102 ms
[12:44:02] Starting 'serve.nodemon'...
[12:44:03] Finished 'serve.nodemon' after 181 ms
[12:44:03] Starting 'serve'...
[12:44:03] Finished 'serve' after 5.38 μs
[12:44:03] [nodemon] 1.8.1
[12:44:03] [nodemon] to restart at any time, enter `rs`
[12:44:03] [nodemon] ignoring: /scratch/universal2/universal/.git/**/* /scratch/universal2/universal/node_modules/**/* /scratch/universal2/universal/bower_components/**/* .sass-cache /scratch/universal2/universal/\.git /scratch/universal2/universal/node_modules/**/node_modules /scratch/universal2/universal/*.js.map /scratch/universal2/universal/*_spec.js /scratch/universal2/universal/angular /scratch/universal2/universal/.DS_Store /scratch/universal2/universal/bower_components/**/* /scratch/universal2/universal/tsd_typings/**/* /scratch/universal2/universal/web_modules/**/* /scratch/universal2/universal/.idea
[12:44:03] [nodemon] watching: /scratch/universal2/universal/dist/**/* /scratch/universal2/universal/index.js
[12:44:03] [nodemon] watching extensions: js,ts,html
[12:44:03] [nodemon] starting `node /scratch/universal2/universal/index.js`
[12:44:03] [nodemon] child pid: 8783
[12:44:03] [nodemon] watching 50 files
module.js:329
    throw err;
    ^

Error: Cannot find module 'preboot'
    at Function.Module._resolveFilename (module.js:327:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/scratch/universal2/universal/dist/modules/universal/server/src/render.js:5:17)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
```

Seems like preboot isn't defined at 'package.json' so I installed it myself: **npm install preboot**

Preboot is successfully installed. Then I try to run again: **npm run serve**

And I got this error:

```
> angular2-universal@0.0.1 serve /scratch/universal2/universal
> gulp serve

[12:47:57] Using gulpfile /scratch/universal2/universal/gulpfile.js
[12:47:57] Starting 'watch'...
[12:47:57] Finished 'watch' after 103 ms
[12:47:57] Starting 'serve.nodemon'...
[12:47:57] Finished 'serve.nodemon' after 209 ms
[12:47:57] Starting 'serve'...
[12:47:57] Finished 'serve' after 5.72 μs
[12:47:57] [nodemon] 1.8.1
[12:47:57] [nodemon] to restart at any time, enter `rs`
[12:47:57] [nodemon] ignoring: /scratch/universal2/universal/.git/**/* /scratch/universal2/universal/node_modules/**/* /scratch/universal2/universal/bower_components/**/* .sass-cache /scratch/universal2/universal/\.git /scratch/universal2/universal/node_modules/**/node_modules /scratch/universal2/universal/*.js.map /scratch/universal2/universal/*_spec.js /scratch/universal2/universal/angular /scratch/universal2/universal/.DS_Store /scratch/universal2/universal/bower_components/**/* /scratch/universal2/universal/tsd_typings/**/* /scratch/universal2/universal/web_modules/**/* /scratch/universal2/universal/.idea
[12:47:57] [nodemon] watching: /scratch/universal2/universal/dist/**/* /scratch/universal2/universal/index.js
[12:47:57] [nodemon] watching extensions: js,ts,html
[12:47:57] [nodemon] starting `node /scratch/universal2/universal/index.js`
[12:47:57] [nodemon] child pid: 8929
[12:47:57] [nodemon] watching 50 files
angular2/core in client: 0.024ms
/scratch/universal2/universal/node_modules/angular2/src/facade/browser.js:4
var win = window;
          ^

ReferenceError: window is not defined
    at Object.<anonymous> (/scratch/universal2/universal/node_modules/angular2/src/facade/browser.js:4:11)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
    at Module.load (module.js:345:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:355:17)
    at require (internal/module.js:13:17)
    at Object.<anonymous> (/scratch/universal2/universal/node_modules/angular2/src/platform/browser/tools/common_tools.js:3:17)
    at Module._compile (module.js:399:26)
    at Object.Module._extensions..js (module.js:406:10)
[12:47:59] [nodemon] app crashed - waiting for file changes before starting...
```

Could you please check it out? Maybe I'm doing something wrong or it's a actual issue.

Thank you
Wallace

**Comment by mcpDESIGNS** [comment]
Duplicate of #164

**Comment by gdi2290** [comment]
The second issue is a problem with our refactor to move package.json into each sub-project (including universal/) rather than merging all build systems. The first issue is a dup of #164 so I'll close this for the other one and update our build system
